src/main/java/com/amazon/ion/impl/macro/Environment.kt

```diff
@@ -21,6 +21,16 @@ data class Environment private constructor(
     val parentEnvironment: Environment?,
 ) {
     fun createChild(arguments: List<Expression>, argumentIndices: List<Int>) = Environment(arguments, argumentIndices, this)
+
+    override fun toString() = """
```

Comment:
🗺️  The only change in this file was adding a custom `toString()` implementation to help when I was debugging things.